code/game/machinery/computer/atmos_computers/__identifiers.dm

```diff
@@ -31,6 +31,9 @@
 #define ATMOS_GAS_MONITOR_DISTRO "distro"
 #define ATMOS_GAS_MONITOR_WASTE "waste"
 
+#define CHAMBER_INPUT_FROM_ID(chamber_id) (chamber_id + "_in")
```

Comment:
Document these both

Also for both of these, wrap chamber_id in parentheses to avoid some precedence shenanigans

Reply:
I think i did this right